app/lib/common/widgets/member_list_entry.dart

```diff
@@ -335,13 +335,21 @@ class MemberListEntry extends ConsumerWidget {
         leading: profile.when(
           data: (data) => ActerAvatar(
             mode: DisplayMode.User,
-            uniqueId: member.userId().toString(),
+            uniqueId: userId,
             size: data.hasAvatar() ? 18 : 36,
             avatar: data.getAvatarImage(),
             displayName: data.displayName,
           ),
           loading: () => const Text('loading'),
-          error: (e, t) => Text('loading avatar failed: $e'),
+          error: (e, t) {
+            debugPrint('loading avatar failed: $e');
+            return ActerAvatar(
+              uniqueId: userId,
+              displayName: userId,
+              mode: DisplayMode.User,
+              size: 18,
```

Comment:
change to size 36 as fallback appears smaller in default avatar size.

Reply:
Fixed by https://github.com/acterglobal/a3/pull/855/commits/823a555785e1664729c8531b7e91e361b42a047f